nss-day-cohort-44/rare-rest-api-fanciful-snowflake
# Ps posts bug fix

# Description
Changed post_time to publication_date in viewsets, models, and fixtures

## Type of change

# Testing Instructions
Check for merge conflicts
1. `git fetch --all`
1. `git checkout ps-posts-bug-fix`
1. `serve`
